frontend/src/store/bucketStore.ts

```diff
@@ -14,8 +20,19 @@ export const useBucketStore = defineStore('bucket', () => {
   async function load() {
     try {
       loading.value = true;
-      const response = await bucketService.searchForBuckets();
-      buckets.value = response.data;
+
+      const identityId = getKeycloak.value.tokenParsed?.idir_user_guid;
```

Comment:
This is going to have to turn into a utils level function - based on the incoming config change work in the other PR, this new utils function will have to iterate through all the defined properties to pull out the identityId from the tokenParsed object. This is fine for now for IDIR only situations, but this can't stay for long.